### PR TITLE
Guests cannot sign-in if Microsoft Teams is disabled on their home te…

### PR DESCRIPTION
**Pull request by premgan**
…nant.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/troubleshoot-guest-access.md [hunk 1]
```diff
@@ -33,7 +33,9 @@ Troubleshoot problems with guest access in Microsoft Teams
 
 ## If your guests are seeing license errors
 
-Guest access in Teams uses Azure Active Directory (Azure AD) Business to Business (B2B) and its licensing model. Guest access is included with all Office 365 Business Premium, Office 365 Enterprise, and Office 365 Education subscriptions. No additional Office 365 license is necessary.
+Guests cannot sign-in if Microsoft Teams is disabled on their home tenant.
```
Comment (JohanFreelancer9) [inline Teams/troubleshoot-guest-access.md]:
```suggestion
Guests cannot sign in if Microsoft Teams is disabled on their home tenant. 
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@somakbhattacharyya please review. Is this change correct?

**Comment by LolaJacobsen** [comment]
I updated this content in a separate PR, so I'm closing this one for now. We retained the existing text and added this note:
> [!NOTE]
> Teams must be enabled on a guest's home tenant for guests to be able to sign in and use Teams as a guest resource on another tenant.

**Pull request closed without merging**